parserator_web/views.py

```diff
@@ -3,7 +3,7 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework.renderers import JSONRenderer
-from rest_framework.exceptions import ParseError
+from rest_framework.exceptions import ParseError # why is this used?
```

Comment:
Good question! If you don't see it referenced, I'd say it's fine to go ahead and remove. 🙃